test/db/tools/rz_bin

```diff
@@ -616,22 +616,22 @@ vaddr      paddr      type   name
 0x080496a8 0x000006a8 SET_32 __libc_start_main
 
 [Fields]
-paddr      name        vaddr      comment    
----------------------------------------------
-0x00000000 ELF         0x00000000 0x464c457f
-0x00000010 Type        0x00000010 0x00000002
-0x00000012 Machine     0x00000012 0x00000003
-0x00000014 Version     0x00000014 0x00000001
-0x00000018 Entry point 0x00000018 0x08048300
-0x0000001c PhOff       0x0000001c 0x00000034
-0x00000020 ShOff       0x00000020 0x000007cc
-0x00000024 Flags       0x00000024 0x00000000
-0x00000028 EhSize      0x00000028 0x00000034
-0x0000002a PhentSize   0x0000002a 0x00000020
-0x0000002c PhNum       0x0000002c 0x00000008
-0x0000002e ShentSize   0x0000002e 0x00000028
-0x00000030 ShNum       0x00000030 0x0000001e
-0x00000032 ShrStrndx   0x00000032 0x0000001b
+paddr      name       vaddr      comment                                         
+---------------------------------------------------------------------------------
+0x00000000 MAGIC      0x00000000 7f 45 4c 46 01 01 01 00 00 00 00 00 00 00 00 00
+0x00000010 Type       0x00000010 0x0002
+0x00000012 Machine    0x00000012 0x0003
+0x00000014 Version    0x00000014 0x00000001
+0x00000018 Entrypoint 0x00000018 0x08048300
+0x0000001c PhOff      0x0000001c 0x00000034
+0x00000020 ShOff      0x00000020 0x000007cc
+0x00000024 Flags      0x00000024 0x0000
```

Comment:
Can we also decode them? See https://github.com/rizinorg/rizin/issues/232 for example.

Reply:
Like `readelf -h`. I am okay, with that idea. But I will implement this on another PR. This one is already too big.

Reply:
Yes, of course. It was a suggestion for further improvements.